OpenRA.Game/Settings.cs

```diff
@@ -88,6 +88,9 @@ public class ServerSettings
 		[Desc("Allow clients to see the country of other clients.")]
 		public bool EnableGeoIP = true;
 
+		[Desc("Record game replays")]
+		public bool RecordGameReplays = false;
```

Comment:
We call them just "Replays" (without the Game) anywhere else, so not needed here?

Reply:
Renamed. Should I update the description to remove the mention of "game" as well?

Reply:
How about expanding this to something like "For dedicated servers only, save replays for all games played."?

Reply:
Thanks, adjusted.